static/mozsearch.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+  <OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
+  <ShortName>SearchFox</ShortName>
+  <Description>SearchFox code search</Description>
+  <InputEncoding>UTF-8</InputEncoding>
+  <Image width="16" height="16" type="image/x-icon">https://searchfox.org/mozilla-central/static/icons/search.png</Image>
+  <Query role="example" searchTerms="BrandFullName" />
+  <Url type="text/html" template="https://searchfox.org/mozilla-central/search?q={searchTerms}&amp;path=&amp;case=false&amp;regexp=false"></Url>
```

Comment:
```suggestion
  <Url type="text/html" template="https://searchfox.org/mozilla-central/search?q={searchTerms}"></Url>
```

These seem unnecessary. Am I overlooking something?

Reply:
For searchfox's caching of dynamic queries it arguably is preferable to use the full encoded form for the extra `&path=&case=false&regexp=false` suffix since the dynamically updated URL from the search form will include the suffix.  That said, because dynamic queries request JSON instead of HTML and the cache `Vary`s on that, it means that this really only matters for the case where 1) the person using OpenSearch looks for a query Foo, 2) some other person types in the search in the form generating an equivalent URL, 3) the person from 2 shares that URL with some other person who accesses it directly.  The person in case 1 will have generated the `Accept: text/html` payload already for the URL shared by the person from case 2, which will generate an instant cache hit for the person in case 3.  (Case 2 would only generate a cache hit for `Accept: application/json`.)